### ISSUE TITLE
There is a problem with the code in gnn_parse.py

### ISSUE THREAD

**Issue by zj12346666** [CharlesShang/DCNv2#135]
attention = torch.sum(torch.softmax(energy, dim=-1), dim=-1)，The result is always 1
